src/Agent/NewRelic/Agent/Core/AgentHealth/AgentHealthReporter.cs

```diff
@@ -92,6 +92,13 @@ public void ReportSupportabilitySummaryMetric(string metricName, float totalSize
             TrySend(metric);
         }
 
+
+        public void ReportSupportabilityGaugeMetric(string metricName, float value)
```

Comment:
This doesn't need to be public yet so we should probably keep this as private for now and also remove it from the interface.